packages/test/test-utils/src/loaderContainerTracker.ts

```diff
@@ -32,6 +33,7 @@ interface ContainerRecord {
 export class LoaderContainerTracker implements IOpProcessingController {
     private readonly containers = new Map<IContainer, ContainerRecord>();
     private lastProposalSeqNum: number = 0;
+    private readonly defaultTimeoutDurationMs: number = 1500;
```

Comment:
The parameter should be required. I don't see a good use case for a default value.